osquery/logger/plugins/kafka_producer.cpp

```diff
@@ -0,0 +1,225 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#include <future>
+#include <memory>
+#include <unistd.h>
```

Comment:
Do you mind placing the C includes above the C++? And include two new lines between the sections?